contracts/src/v0.8/ccip/FeeQuoter.sol

```diff
@@ -493,6 +495,14 @@ contract FeeQuoter is AuthorizedCallers, IFeeQuoter, ITypeAndVersion, IReceiver,
     }
   }
 
+  /// @notice Signals which version of the pool interface is supported
+  function supportsInterface(
```

Comment:
should we not use 

`interfaceId == type(IFeeQuoter).interfaceId || super.supportsInterface` or something similar?